mojo/services/keyboard/interfaces/BUILD.gn

```diff
@@ -9,4 +9,10 @@ mojom("interfaces") {
   sources = [
     "keyboard.mojom",
   ]
+
+  import_dirs = [ get_path_info("../../", "abspath") ]
+
+  deps = [
+    "../../native_viewport/interfaces",
```

Comment:
Compare with https://codereview.chromium.org/1453823005/patch/1/10003 which duplicates this change by copy pasta...